src/xrc/xh_menu.cpp

```diff
@@ -79,6 +80,18 @@ wxObject *wxMenuXmlHandler::DoCreateResource()
             wxString label = GetText(wxT("label"));
 #if wxUSE_ACCEL
             wxString accel = GetText(wxT("accel"), false);
+            wxVector<wxString> extra_accels;
+            if (HasParam(wxT("extra-accels")))
+            {
+                wxXmlNode* extra_accels_node = GetParamNode(wxT("extra-accels"));
+                wxXmlNode* extra_accel_node = extra_accels_node->GetChildren();
```

Comment:
A few things here:

- We use `camelCase` in wx.
- Local variables should be `const` when they don't change.
- Having two variables differing by a single letter should be avoided if possible.

So I'd like to change this to something like this instead:
```suggestion
                wxXmlNode* const extraAccelsNode = GetParamNode(wxT("extra-accels"));
                wxXmlNode* node = extraAccelsNode->GetChildren();
```